### PR TITLE
10372 automated accessibility test using axe playwright

### PR DESCRIPTION
**Pull request by kajalkundu**
This pull request for automated accessibility tests using axe playwright and jest framework.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "@types/jest": "^26.0.23",
     "@types/jest-axe": "^3.5.1",
```
Comment (gregzealley) [inline package.json:50]: Can jest-axe be removed?

File: Tests/FunctionalTests/fss-home.spec.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ const { autoTestConfig } = require('./appSetting');
 const {pageObjectsConfig} = require('./pageObjects');
 
 describe('Test Home Page Scenario', () => {
-    jest.setTimeout(120000);
+    jest.setTimeout(30000);
```
Comment (gregzealley) [inline Tests/FunctionalTests/fss-home.spec.ts:6]: Worth having this value as a constant?

**Review by gregzealley — CHANGES_REQUESTED** [review]

**Review by gregzealley — APPROVED** [review]

**Review by gregzealley — APPROVED** [review]

**Review by bhalcha13813 — APPROVED** [review]
Looks good to me

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**